README.md

```diff
@@ -33,21 +33,26 @@ Having a bug tracking system doesn't make bug handling efficient and CI/CD doesn
 # Tech
 
 - What are the usual stacks used at the company?
-- How do you use source control?
+- Is there default operating system used or is it up to employees to choose one?
+- How do you use source control? Is it git/svn?
```

Comment:
Thoughts about re-phrasing this to be more general?
```
What do you use for source control and how do you use it?
```

Reply:
👍 
Maybe instead of `how do you use it?` ask if they use any branching strategies or practices.

Reply:
Good point, I've added proposed change + reference to example of branching strategy 😸